### PR TITLE
Use newer version of Jgroups and EclipseLink to prevent PKIX errors on OCP 4.x

### PR DESCRIPTION
**Pull request by mshaposhnik**
### What does this PR do?

Use newer version of Jgroups, KUBE_PING and EclipseLink to prevent PKIX errors on OCP 4.x

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13607

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/jgroups/che-tcp.xml [hunk 1]
```diff
@@ -21,29 +21,30 @@
...
+    log_discard_msgs="false"
     thread_naming_pattern="cl"
 
     thread_pool.min_threads="0"
     thread_pool.max_threads="500"
     thread_pool.keep_alive_time="30000" />
 
-  <kubernetes.KUBE_PING
-    labels="app=che" />
+  <org.jgroups.protocols.kubernetes.KUBE_PING
```
Comment (skabashnyuk) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/jgroups/che-tcp.xml:31]: What the source of this file? Does it come from documentation

**Review by skabashnyuk — COMMENTED** [review]
in general looks good.

**Comment by mshaposhnik** [comment]
ci-build

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/jgroups/che-tcp.xml (hunk 1 above)
Comment (mshaposhnik) [reply assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/jgroups/che-tcp.xml:31]: Yes that's exacly as in doc on https://github.com/jgroups-extras/jgroups-kubernetes

**Review by skabashnyuk — APPROVED** [review]

**Comment by mshaposhnik** [comment]
ci-test

**Comment by mshaposhnik** [comment]
ci-test

**Comment by mshaposhnik** [comment]
crw-ci-test

**Comment by mshaposhnik** [comment]
crw-ci-test

**Comment by mshaposhnik** [comment]
Merging since the same test failure is in master (and no other regressions visible)

**Pull request merged**